components/common/comment_icon.tsx

```diff
@@ -33,7 +33,7 @@ export default class CommentIcon extends React.PureComponent<Props> {
         if (this.props.commentCount > 0) {
             iconStyle += ' post-menu__item--show';
             commentCountSpan = (
-                <span className='post-menu__comment-count'>
+                <span className='post-menu__comment-count unselectable'>
```

Comment:
Instead of marking things with a helper class, is it maybe better form to add `user-select: none` to the classes that are already on these components? 0/5, but maybe people with more experience with CSS would have a stronger opinion on that.

Reply:
Yes the original solution was to use “user-select”, like it’s in the first commit. This helper class was added to help exclude those elements from the selection because document.getSelection() doesn’t take the css selector into account and was copying them regardless. I will revert to that solution since we won’t be modifying the selection anymore